RecoTracker/TrackProducer/plugins/TrackExtraRekeyer.cc

```diff
@@ -0,0 +1,62 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "DataFormats/TrackReco/interface/TrackExtraFwd.h"
+
+class TrackExtraRekeyer : public edm::stream::EDProducer<> {
```

Comment:
in #35620 this is a global module.
Please apply a more verbatim port.

Reply:
I tried that first but that didn't compile...

Reply:
`edm::global` certainly exists in 10_6_X.
What was the compilation error?

Reply:
```
/afs/cern.ch/work/t/tvami/public/HSCP/Misc/RefittingBackport/CMSSW_10_6_X_2021-10-31-2300/src/RecoTracker/TrackProducer/plugins/TrackExtraRekeyer.cc:8:57: error: expected template-name before '<' token
 class TrackExtraRekeyer : public edm::global::EDProducer<> {
                                                         ^
/afs/cern.ch/work/t/tvami/public/HSCP/Misc/RefittingBackport/CMSSW_10_6_X_2021-10-31-2300/src/RecoTracker/TrackProducer/plugins/TrackExtraRekeyer.cc:8:57: error: expected '{' before '<' token
/afs/cern.ch/work/t/tvami/public/HSCP/Misc/RefittingBackport/CMSSW_10_6_X_2021-10-31-2300/src/RecoTracker/TrackProducer/plugins/TrackExtraRekeyer.cc:8:57: error: expected unqualified-id before '<' token
```

Reply:
resolved in https://github.com/cms-sw/cmssw/pull/36001/commits/ccd949047995327cead47f239fb8b664ebbe40c8